update to use maven wrapper

gkwan-ibm opened the pull request:
address issue https://github.com/OpenLiberty/guides-common/issues/116

gkwan-ibm committed:
Merge pull request #231 from OpenLiberty/staging

Merge staging to prod - Dependabot Bump

gkwan-ibm committed:
update to use maven wrapper

Signed-off-by: Gilbert Kwan <gkwan@ca.ibm.com>

gkwan-ibm committed:
update to use maven wrapper

Signed-off-by: Gilbert Kwan <gkwan@ca.ibm.com>

gkwan-ibm committed:
update to use maven wrapper

Signed-off-by: Gilbert Kwan <gkwan@ca.ibm.com>

gkwan-ibm committed:
update to use maven wrapper

Signed-off-by: Gilbert Kwan <gkwan@ca.ibm.com>

gkwan-ibm committed:
update to use maven wrapper

Signed-off-by: Gilbert Kwan <gkwan@ca.ibm.com>

gkwan-ibm committed:
update to use maven wrapper

Signed-off-by: Gilbert Kwan <gkwan@ca.ibm.com>

gkwan-ibm committed:
Update README.adoc

dev0508 commented:
Everything works fine on linux but in README "docker-compose" has to be changed to "docker compose"

gkwan-ibm commented:
> Everything works fine on linux but in README "docker-compose" has to be changed to "docker compose"

hi @dev0508, `docker-compose` is a command. Why need to change to "docker compose"?

dev0508 commented:
doc

> > Everything works fine on linux but in README "docker-compose" has to be changed to "docker compose"
> 
> hi @dev0508, `docker-compose` is a command. Why need to change to "docker compose"?

`docker-compose` doesn't work anymore they have shifted to use `docker compose` 
https://docs.docker.com/compose/gettingstarted/ 
in the above link if you scroll to Step 3 they have used docker compose as well for their example

GeeTransit reviewed (approved):
Tested successfully on MacOS

AndrewSasmito reviewed (approved):
Tested on cloud, no issues

GeeTransit reviewed (approved):
Tested successfully on Windows. Both `docker-compose` and `docker compose` work for deploying pact-broker

dev-koan committed:
Update README to use
Usage:  docker compose [OPTIONS] COMMAND

Define and run multi-container applications with Docker

Options:
      --all-resources              Include all resources, even those not used by services
      --ansi string                Control when to print ANSI control characters ("never"|"always"|"auto") (default "auto")
      --compatibility              Run compose in backward compatibility mode
      --dry-run                    Execute command in dry run mode
      --env-file stringArray       Specify an alternate environment file
  -f, --file stringArray           Compose configuration files
      --parallel int               Control max parallelism, -1 for unlimited (default -1)
      --profile stringArray        Specify a profile to enable
      --progress string            Set type of progress output (auto, tty, plain, json, quiet) (default "auto")
      --project-directory string   Specify an alternate working directory
                                   (default: the path of the, first specified, Compose file)
  -p, --project-name string        Project name

Commands:
  attach      Attach local standard input, output, and error streams to a service's running container
  build       Build or rebuild services
  commit      Create a new image from a service container's changes
  config      Parse, resolve and render compose file in canonical format
  cp          Copy files/folders between a service container and the local filesystem
  create      Creates containers for a service
  down        Stop and remove containers, networks
  events      Receive real time events from containers
  exec        Execute a command in a running container
  export      Export a service container's filesystem as a tar archive
  images      List images used by the created containers
  kill        Force stop service containers
  logs        View output from containers
  ls          List running compose projects
  pause       Pause services
  port        Print the public port for a port binding
  ps          List containers
  pull        Pull service images
  push        Push service images
  restart     Restart service containers
  rm          Removes stopped service containers
  run         Run a one-off command on a service
  scale       Scale services
  start       Start services
  stats       Display a live stream of container(s) resource usage statistics
  stop        Stop services
  top         Display the running processes
  unpause     Unpause services
  up          Create and start containers
  version     Show the Docker Compose version information
  wait        Block until containers of all (or specified) services stop.
  watch       Watch build context for service and rebuild/refresh containers when files are updated

Run 'docker compose COMMAND --help' for more information on a command.

Signed-off-by: Dev Agarwal <devagarwal0508@gmail.com>

gkwan-ibm committed:
Update README.adoc
